source/developers/quickstarts/first_openedx_pr.rst

```diff
@@ -175,38 +158,127 @@ To have Tutor run your local fork of edx-platform, you have to tell it to do so
+   http://apps.local.openedx.io:1997/account
+   http://apps.local.openedx.io:2000/learning
+   http://apps.local.openedx.io:2002/discussions
 
-From this point on, whatever changes you make to the code in your clone should
-be visible in your local LMS instance immediately.
+These ports and paths are to specific MFEs made available via the MFE plugin.
+
+In order to develop locally, you will need to fork and clone the MFE
+repository as you did for edx-platform, bind mount the directory, stop
+the Tutor-hosted MFE and start a local npm dev server.  Let's do so
+with the Learner Dashboard MFE.
+
+First, you should verify that the learner dashboard is working
+correctly after you have installed the MFE plugin.  Assuming
+everything is configured in the standard way, your URL should be
+``http://apps.local.openedx.io:1996/learner-dashboard/``
+
+Follow the same, fork, clone workflow described above and clone the
+learner-dashboard
+``https://github.com/openedx/frontend-app-learner-dashboard``
+repository locally.
+
+Add a tutor mount for your cloned directory.
+
+.. code-block:: bash
+
+   (tutor-main) $ tutor mounts add /home/git/frontend-app-learner-dashboard
+
+Next, ensure that the learner-dashboard MFE is stopped
+
+.. code-block:: bash
+
+   (tutor-main) $ tutor dev stop learner-dashboard
+
+Reloading the learner dashboard page in the browser should now yield an error.
+
+Now its time to replace the default Tutor-hosted learner-dashboard with a
+local version.  That version will use a dev config file to connect to
+the Tutor-hosted backend and to bind to the expected port.
+
+.. admonition:: Local MFE Support and npm dev Profiles
+
+   Not every MFE currently has an ``npm run dev`` command that will
+   work with Tutor, though it is possible to create one if that is the
+   case for the MFE you are developing, , using `an existing one
+   <https://github.com/openedx/frontend-app-learner-dashboard/pull/530/files>`_
+   as a template.
+
+From the directory containing the local copy of the learner-dashboard
+repository start the npm dev server.
+
+.. code-block:: bash
+
+   (tutor-main) $ npm run dev
 
 Exercise: Update the Learner Dashboard
 **************************************
 
-The Learner Dashboard is the first page that students will see when they log
-into Open edX. On our Tutor dev environment, it is located at
+The Learner Dashboard is the first page that students will see after they log
+into Open edX.
+
+The dashboard page will either be provided by a legacy server side template or by the learner-dashboard MFE.  Working with the MFE is usually the best choice.  Note that this tutorial is simplistic and appropriate to get started.  However, we don't recommend forking an MFE to customize it.  For extensive modifications, that might be necessary, but for simple things using design tokens and frontend plugin slots will be a much better alternative.
+
+Working with the Learner Dashboard MFE
+======================================
+
+
```

Comment:
@arbrandes thoughts about a change that makes sense to kick the tires?  I updated a string in messages.json which I didn't think we great, it would be better to change a little react code I think.